athena-federation-sdk/pom.xml

```diff
@@ -216,8 +222,8 @@
         </dependency>
         <dependency>
             <groupId>org.bouncycastle</groupId>
-            <artifactId>bcprov-jdk15on</artifactId>
-            <version>1.70</version>
+            <artifactId>bcprov-jdk18on</artifactId>
+            <version>1.78.1</version>
```

Comment:
Can you please check if bouncy castle are part of shaded jar? if no, can we included it in and relocate?

Reply:
Bouncy Castle was already included in the shaded JAR; I have now relocated the class path to prevent any conflicts.